packages/client/src/Collection.ts

```diff
@@ -84,15 +84,20 @@ export class Collection<T> {
 
   get = async (): Promise<CollectionList<T>> => {
     const res = await this.client.request({
-      url: `/collections/${encodeURIComponent(this.id)}/documents`,
+      url: `/collections/${encodeURIComponent(this.id)}/records`,
       method: 'GET',
     }).send()
 
     return res.data
   }
 
-  doc = (id: string): Doc<T> => {
-    return new Doc<T>(id, this, this.client, this.onDocSnapshotRegister)
+  record = (id: string): CollectionRecord<T> => {
+    return new CollectionRecord<T>(id, this, this.client, this.onCollectionRecordSnapshotRegister)
+  }
+
+  // Deprecated
```

Comment:
We could use the @deprecated jsdoc annotation so editors like VS Code show this function with a strikethrough if you use it.
https://jsdoc.app/tags-deprecated.html